lib/ansible/modules/cloud/amazon/rds_cluster.py

```diff
@@ -0,0 +1,843 @@
+#!/usr/bin/python
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
+---
+module: rds_cluster
+short_description: Manages RDS Clusters for Aurora instances
+description:
+  -Manages the creation, modification, deletion, snapshot and restore of RDS Clusters for Aurora instances in AWS
+version_added: "2.2"
+author: "Nick Aslanidis (@naslanidis)"
+options:
+  command:
+    description:
+      - specifies the action to take
+      - absent to remove resource
+    required: true
+    default: None
+    choices: [ 'create', 'delete', 'modify', 'snapshot', 'restore' ]
```

Comment:
Instead of command this module should use `state=` and present/absent. Having a `command=` like this means playbooks won't be idempotent. Also the docs don't match the choices specified.